### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by carodelafuente** [garmendiabenjamin/prj-rev-bwfs-dasmoto#1]
Try getting in the habit of always writing an `alt` attribute in your img tags. It's considered best practices when writing html that is accessible for all users. In this case, someone might be using a screen reader or maybe their browser failed to load the image, it's a good fallback to have some replacement text.
